src/osd/OSD.cc

```diff
@@ -10146,6 +10146,17 @@ void OSD::ShardedOpWQ::_process(uint32_t thread_index, heartbeat_handle_d *hb)
     sdata->sdata_op_ordering_lock.Unlock();
     return;    // OSD shutdown, discard.
   }
+  if (item.second.is_future()) {
```

Comment:
Could this get formatted better.

Reply:
replace `is_future` function with `future_op`,  did you mean it ?